src/main/java/org/api/entity/Category.java

```diff
@@ -0,0 +1,40 @@
+package org.api.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Entity
+@Data
+@AllArgsConstructor
+@Table(name = "Category" ,schema = "goal")
+public class Category {
+
+    @Id
+    private  Integer id;
+
+    @Column(name = "category_name")
+    private String category_name;
```

Comment:
Please follow camelCase naming convention for all properties

Reply:
Done